components/widgets/CurrentData/patterns/Windspeed/index.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from "prop-types";
+import { windspeedUnitType } from "@/components/shapes/units";
+import { useTranslation } from "react-i18next";
+import WidgetBackground from "@/components/atomic/WidgetBackground";
+import * as Styled from "../../styles";
+
+const WindspeedCurrent = ({ windspeed = 0, unit = "m" }) => {
+  const {
+    t,
+    i18n: { language = "en" },
+  } = useTranslation();
+
+  return (
+    <WidgetBackground>
+      <Styled.Label>
+        {t("summit_dashboard.widgets.current.windspeed")}
+      </Styled.Label>
+      <Styled.Value $variant="large">
+        {new Intl.NumberFormat(language, {
```

Comment:
doesn't need a unit formatter since we're not showing the value and unit together, but as a value with label.